### PR TITLE
[GDI32_APITEST] Improve SetWorldTransform testcase

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose
Improve the testcase for `gdi32!SetWorldTransform` function.
JIRA issue: [CORE-15554](https://jira.reactos.org/browse/CORE-15554)

### CODE REVIEW

**Comment by katahiromz** [comment]
Win10:
SetWorldTransform-Win10
Successful.
Win2k3:
SetWorldTransform-Win2k3
Successful.

File: modules/rostests/apitests/gdi32/SetWorldTransform.c [hunk 1]
```diff
@@ -70,15 +71,15 @@ void Test_SetWorldTransform()
...
+    ok(xform.eM21 == 4, "xform.eM21 should be 4\n");
+    ok(xform.eM22 == 6, "xform.eM22 should be 6\n");
 
     /* Set smallest possible values */
-    xform.eM11 = (FLOAT)1.4e-45;
+    xform.eM11 = 1.17549435e-38f;
     xform.eM12 = 0;
     xform.eM21 = 0;
-    xform.eM22 = (FLOAT)1.4e-45;
+    xform.eM22 = 1.17549435e-38f;
```
Comment (HBelusca) [inline modules/rostests/apitests/gdi32/SetWorldTransform.c]: What happens if one uses the older values?

File: modules/rostests/apitests/gdi32/SetWorldTransform.c [hunk 2]
```diff
@@ -89,9 +90,30 @@ void Test_SetWorldTransform()
...
+    ok_int(result, 0);
+
+    result = GetWorldTransform(hdc, &xform);
+    ok_int(result, 1);
+    ok(xform.eM11 == 0, "xform.eM11 should be 0\n");
+    ok(xform.eM12 == 1, "xform.eM12 should be 1\n");
+    ok(xform.eM21 == 1, "xform.eM21 should be 1\n");
+    ok(xform.eM22 == 0, "xform.eM22 should be 0\n");
 
 	DeleteDC(hdc);
```
Comment (HBelusca) [inline modules/rostests/apitests/gdi32/SetWorldTransform.c]: Indentation may need a fix there.

File: modules/rostests/apitests/gdi32/SetWorldTransform.c (hunk 1 above)
Comment (katahiromz) [reply modules/rostests/apitests/gdi32/SetWorldTransform.c]: Fails in Win2k3.

File: modules/rostests/apitests/gdi32/SetWorldTransform.c (hunk 2 above)
Comment (katahiromz) [reply modules/rostests/apitests/gdi32/SetWorldTransform.c]: OK.

**Pull request merged**